### PR TITLE
first step for anagram exercise

### PR DESCRIPTION
**Pull request by jaggededgedjustice**

### CODE REVIEW

**Comment by kotp** [comment]
Thank you for tackling this @jaggededgedjustice .  The `config.json` does not include `anagram` but will need it to pass the Travis task.  Can you make the needed change?

**Comment by jaggededgedjustice** [comment]
Yes, I'll take care of that after I add all the tests and make sure the example works

File: exercises/anagram/example.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  exit 1
+fi
+
+input="$1"
+posibles="$2"
+
+out=""
+
+in_counts="$(echo $input | tr '[:upper:]' '[:lower:]' | fold -w1 | sort | uniq -c)"
+for posible in $posibles
```
Comment (kotp) [inline exercises/anagram/example.sh]: `$posibles` should have three "s"'s.  Two in the middle one on the end.

**Comment by kotp** [comment]
Thank you @jaggededgedjustice for getting this in.

**Pull request merged**